experiment/kind-detect-local-e2e.sh

```diff
@@ -54,12 +54,18 @@ signal_handler() {
 }
 trap signal_handler INT TERM
 
-# build kubernetes / node image, e2e binaries
+# build kubernetes / node image, e2e binaries and ginkgo
 build() {
   # build the node image w/ kubernetes
   kind build node-image -v 1
+  # Ginkgo v1 is used by Kubernetes 1.24 and earlier, fallback if v2 is not available.
+  GINKGO_V2_SRC="vendor/github.com/onsi/ginkgo/v2/ginkgo"
+  GINKGO_TARGET=${GINKGO_V2_SRC#"vendor/"}
+  if [ ! -d "$GINKGO_V2_SRC" ]; then
+      GINKGO_TARGET="vendor/github.com/onsi/ginkgo/ginkgo"
+  fi
```

Comment:
I find that unnecessarily complex. How about this:
```suggestion
  # Ginkgo v1 is used by Kubernetes 1.24 and earlier and exists
  # in the vendor directory. Historically it has been built
  # with the "vendor" prefix.
  GINKGO_TARGET="vendor/github.com/onsi/ginkgo/ginkgo"
  if [ ! -d "$GINKGO_TARGET" ]; then
      # If the directory doesn't exist, then we must be on Kubernetes
      # >= 1.25 with Ginkgo V2. The "vendor" prefix is no longer
      # needed.
      GINKGO_TARGET="github.com/onsi/ginkgo/v2/ginkgo"
  fi
```

Reply:
Done.